optics-vl/src/Optics/VL.hs

```diff
@@ -168,3 +216,33 @@ withPrismVL
   -> r
 withPrismVL o k = k (toPrismVL o)
 {-# INLINE withPrismVL #-}
+
+----------------------------------------
+
+type GetterVL s a =
+  forall f. (Contravariant f, Functor f) => (a -> f a) -> s -> f s
+
+-- | Build a 'Getter' from the van Laarhoven representation.
+getterVL :: GetterVL s a -> Getter s a
+getterVL o = to (getConst #. o Const)
+{-# INLINE getterVL #-}
+
+-- | Convert a 'Getter' to the van Laarhoven representation.
+toGetterVL :: Is k A_Getter => Optic' k is s a -> GetterVL s a
+toGetterVL o = unwrapProfunctor #. getOptic (castOptic @A_Getter o) .# WrapProfunctor
+{-# INLINE toGetterVL #-}
+
+----------------------------------------
+
+type FoldVL s a =
+  forall f. (Contravariant f, Applicative f) => (a -> f a) -> s -> f s
+
+-- | Build a 'Fold' from the van Laarhoven representation.
+foldVL :: FoldVL s a -> Fold s a
+foldVL o = O.foldVL $ \f -> runTraversed . getConst #. o (Const #. Traversed #. f)
```

Comment:
This looks great. But how to rename `foldVL` we already have. I don't have any ideas.

Reply:
foldVL used to be named `mkFold`. How about that?